docs/podman-exec.1.md

```diff
@@ -38,6 +38,14 @@ Sets the username or UID used and optionally the groupname or GID for the specif
 The following examples are all valid:
 --user [user | user:group | uid | uid:gid | user:gid | uid:group ]
 
+**-w**, **--workdir**=""
```

Comment:
Can you flip the order here?  `**--workdir**, **-w**=""`  We generally list the long form of the option first, the short form second.

Reply:
Done.

Looks like a few other man pages (eg., podman-create.1, podman-run.1) also have the long and short variants inverted.

Reply:
Lets clean them up in a separate man page scrub PR.

Reply:
Ack.

Reply:
Filed #2103 for this.

Reply:
I've created #2104 so we don't forget about it.  @debarshiray thanks for the heads up!